### PR TITLE
[10.0][FIX] account_financial_report_qweb: css injection assets backend

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
#430

### CODE REVIEW

**Comment by pedrobaeza** [comment]
No more is needed?

Please change module version number

**Comment by HviorForgeFlow** [comment]
I don't think so, it shouldn't show css conflicts anymore.

File: account_financial_report_qweb/view/report_template.xml [hunk 1]
```diff
@@ -4,7 +4,6 @@
     <template id="account_financial_report_assets_backend"
               name="account_financial_report assets" inherit_id="web.assets_backend">
         <xpath expr="." position="inside">
-            <link href="/account_financial_report_qweb/static/src/css/report.css" rel="stylesheet"/>
```
Comment (yajo) [inline account_financial_report_qweb/view/report_template.xml:7]: AFAICS ths file is not being used anymore, so you could remove it, right?

Or, if it is still needed for reports, then move it to some `assets_report` view.

**Review by yajo — CHANGES_REQUESTED** [review]

File: account_financial_report_qweb/view/report_template.xml (hunk 1 above)
Comment (HviorForgeFlow) [reply account_financial_report_qweb/view/report_template.xml:7]: It is used here: https://github.com/OCA/account-financial-reporting/blob/25e705cf833ac68bc3630d69ca2934f1e08de987/account_financial_report_qweb/report/templates/layouts.xml#L4-L8

**Review by yajo — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Thanks for this. Please forward-port it to v11.

**Comment by pedrobaeza** [comment]
I think this has broken the view in screen option. Can it be?

**Pull request merged**